### PR TITLE
Provide class which extends SpringBootApplication

### LINKED ISSUE #1: Provide class which extends SpringBootApplication

### PR DESCRIPTION
**Pull request by mrcombajn**
https://github.com/mrcombajn/spring-training-app/issues/1

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -7,10 +7,48 @@
...
+                <type>import</type>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+            <version>3.1.1</version>
```
Comment (greenwind) [inline pom.xml]: I don't think version is necessary here

File: src/main/java/pl/trainingapp/controller/HomeController.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package pl.trainingapp.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
```
Comment (greenwind) [inline src/main/java/pl/trainingapp/controller/HomeController.java]: You can replace @Controller and @ResponseBody with @RestController

**Review by greenwind — DISMISSED** [review]

File: src/main/java/pl/trainingapp/controller/HomeController.java (hunk 2 above)
Comment (mrcombajn) [reply src/main/java/pl/trainingapp/controller/HomeController.java]: Replacer @Controller with @RestController

**Review by greenwind — DISMISSED** [review]

File: pom.xml (hunk 1 above)
Comment (mrcombajn) [reply pom.xml]: fixed it!

**Review by greenwind — APPROVED** [review]

**Pull request merged**